meshtastic/message.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package meshtastic;
+
+/*
+ * Representation of a received or sent message to be stored long-term to create a
+ * message history
+ */
+message Message {
+  /*
+   * The sending node's short name.
+   */
+  string sender_short_name = 1;
```

Comment:
Please use the nodenum instead

Reply:
This field is used for displaying a device's name on the screen. I don't think it'd be user-friendly to show the device number on the screen rather than the short name.

Reply:
i was looking at it from a database point of view, and the short name is not a good way to provide a unique key. Maybe i don'tr fully get how the link to the user entries is made, especially if it is a DM

Reply:
No link to the user entry is made given that that's not needed to display the name.